README.adoc

```diff
@@ -2198,6 +2198,19 @@ something = something && something.downcase
 something &&= something.downcase
 ----
 
+=== Equal Object Identifier [[equal-object-id]]
```

Comment:
I think we can name this "Identity Comparison", as the other form of comparison is "Value Comparison" (the one normally done with `==`).